src/diagnostic-channel-publishers/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "diagnostic-channel-publishers",
   "version": "0.1.3",
   "main": ".dist/src/index.js",
-  "typings": ".dist/src/index.d.ts",
+  "types": ".dist/src/index.d.ts",
```

Comment:
So I thought this was the cause of an issue that I was having but it turned out to be unrelated. With that being said, the documentation says that 'types' and 'typings' are equivalent, so I can easily revert this if that's preferable.